Gopkg.toml

```diff
@@ -0,0 +1,582 @@
+# Gopkg.toml example
```

Comment:
We've had several recent dependency.tsv updates, please ensure this file reflects all those.

Reply:
Yes, I just double checked this morning.

tsvSha  matchOrNot  depName
3b1ae45394a234c385be014e9a488f2bb6eef821	True	cloud.google.com/go
03719f665513a71a89cf687d260c39a120ac00e7	True	github.com/Azure/azure-sdk-for-go
10cfe58defab0c9a33be1f7b3ee656857670b509	True	github.com/Azure/go-autorest
4e5a4a63d9b78757b4891b8de60be2606d8972ee	True	github.com/EvilSuperstars/go-cidrman
89e3ac64b5b3e403a5e7c35ea4f98d45db7b4518	True	github.com/ajstarks/svgo
31228935eec685587914528585da4eb9b073c76d	True	github.com/altoros/gosigma
93f237eba9b0602f3e73710416558854a81d9337	True	github.com/armon/go-metrics
3ac7bf7a47d159a033b107610db8a1b6575507a4	True	github.com/beorn7/perks
c068ca2f0aacee5ac3681d68e4d0a003b7d1fd2c	True	github.com/bmizerany/pat
e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd	True	github.com/boltdb/bolt
666e3beca3cb4f6b5c8f176ba80daf2b3adbd84e	True	github.com/cloud-green/monitoring
7b2428fec40033549c68f54e26e89e7ca9a9ce31	True	github.com/coreos/go-systemd
01aeca54ebda6e0fbfafd0a524d234159c05ec20	True	github.com/dgrijalva/jwt-go
145fabdb1ab757076a70a886d092a3af27f66f4c	True	github.com/dustin/go-humanize
f0cc927784781fa395c06317c58dea2841ece3a9	True	github.com/docker/distribution
32c6cc29c14570de4cf6d7e7737d68fb2d01ad15	True	github.com/godbus/dbus
44145f04b68cf362d9c4df2182967c2275eaefed	True	github.com/golang/glog
69521b3833175dfcfb1cc1fdb0c9be92e66faa81	True	github.com/golang/mock
b4deda0973fb4c70b50d226b1af49f3da59f5265	True	github.com/golang/protobuf
9235644dd9e52eeae6fa48efd539fdc351a0af53	True	github.com/google/go-querystring
44d81051d367757e1c7c6a5a86423ece9afcf63c	True	github.com/google/gofuzz
13d73096a474cac93275c679c7b8a2dc17ddba82	True	github.com/gorilla/handlers
08023a0215e7fc27a9aecd8b8c50913c40019478	True	github.com/gorilla/schema
5ed622c449da6d44c3c8329331ff47a9e5844f71	False	github.com/gorilla/websocket
36ee7e946282a3fb1cfecd476ddc9b35d8847e42	True	github.com/gosuri/uitable
fa3f63826f7c23912c15263591e65d54d080b458	True	github.com/hashicorp/go-msgpack
077966dbc90f342107eb723ec52fdb0463ec789b	True	github.com/hashicorp/raft
6e5ba93211eaf8d9a2ad7e41ffad8c6f160f9fe3	True	github.com/hashicorp/raft-boltdb
ade826b8b54e81a779ccb29d358a45ba24b7809c	True	github.com/joyent/gocommon
2f11feadd2d9891e92296a1077c3e2e56939547d	True	github.com/joyent/gosdc
0da0d5f1342065321c97812b1f4ac0c2b0bab56c	True	github.com/joyent/gosign
b99631de12cf04a906c1d4e4ec54fb86eae5863d	True	github.com/juju/ansiterm
bc5b7e5db16683db8ddffa7eef67b53706a4618a	True	github.com/juju/bundlechanges
e74f39857ca013cf63947ba2843806f7afdd380d	True	github.com/juju/cmd
9be91dc79b7c185fa8b08e7ceceee40562055c83	True	github.com/juju/collections
25349c35a6b1c9244c24045c9b5faab5932eae63	True	github.com/juju/description
1b5e39b83d1835fa480e0c2ddefb040ee82d58b3	False	github.com/juju/errors
2ce1bb71843d6d179b3f1c1c9cb4a72cd067fc65	True	github.com/juju/gnuflag
932a8cea00a1cd5cba3829a672c823955db674ae	True	github.com/juju/go-oracle-cloud
40d72ab9641a2a8c36a9c46a51e28367115c8e59	True	github.com/juju/go4
afe8b77aa08f272b49e01b82de78510c11f61500	True	github.com/juju/gojsonpointer
f0d24ac5ee330baa21721cdff56d45e4ee42628e	True	github.com/juju/gojsonreference
e1ad140384f254c82f89450d9a7c8dd38a632838	True	github.com/juju/gojsonschema
abe11904dd8cd40f0777b7704ae60348a876542e	True	github.com/juju/gomaasapi
14bf14c307672fd2456bdbf35d19cf0ccd3cf565	True	github.com/juju/httpprof
266fd1e9debf09c037a63f074d099a2da4559ece	True	github.com/juju/httprequest
fb1dc7175251ac8e2dc460897313e05175cbbc0d	True	github.com/juju/idmclient
a0ef8b74ebcffeeff9fc374854deb4af388f037e	True	github.com/juju/jsonschema
8232ab8918d91c72af1a9fb94d3edbe31d88b790	True	github.com/juju/loggo
24974d6c264fe5a29716e7d56ea24c4bd904b7cc	True	github.com/juju/mempool
d21b13acf4bfd8a8b0482a3a78e44d98880b40d3	True	github.com/juju/mutex
5b81707e882b8293e6cf77854b4cd49f29776e11	True	github.com/juju/naturalsort
325989f58c78b64db349b60742495b48e5468ec1	True	github.com/juju/os
ba21344fff207f4fa06b0a08bac386b179a2e6a1	True	github.com/juju/packaging
d67418f14c93a698e37b52468958d5d4dcf8a7dd	True	github.com/juju/persistent-cookiejar
5f8741c297b47cba29f747c67f2e31e7c2a36ecb	True	github.com/juju/proxy
091412f84753115324ce870066ffbb6bb55c4ef4	True	github.com/juju/pubsub
5b9ff866471762aa2ab2dced63c9fb6f53921342	True	github.com/juju/ratelimit
86037679224858cd82666e4877a8ac4c42cbda8b	True	github.com/juju/replicaset
62c62032529169c7ec02fa48f93349604c345e1f	True	github.com/juju/retry
b058ad085c9480f110200bfe2309189aff5780b8	True	github.com/juju/rfc
958d8b71e5ffa51212133b12a998ddf8362739df	True	github.com/juju/romulus
075de04f9b7d7580d60a1e12a0b3f50bb18e6998	True	github.com/juju/schema
9b925afd677234e4146dde3cb1a11e187cbed64e	True	github.com/juju/terms-client
72703b1e95eb8ce4737fd8a3d8496c6b0be280a6	False	github.com/juju/testing
f50b17d1ff3c31b7ea1c70e0d38a83f19af6d334	True	github.com/juju/txn
68a59c96c178fbbad65926e7f93db50a2cd14f33	True	github.com/juju/usso
c746c6e86f4fb2a04bc08d66b7a0f7e900d9cbab	True	github.com/juju/utils
1f41e27e54f21acccf9b2dddae063a782a8a7ceb	True	github.com/juju/version
54b8c57083b4afb7dc75da7f13e2967b2606a507	True	github.com/juju/webbrowser
eb759a627588d35166bc505fceb51b88500e291e	True	github.com/juju/xml
f6b1e93fa2e29a1d7d49b566b2b51efb060c982a	True	github.com/juju/zip
77a895ad01ebc98a4dc95d8355bc825ce80a56f6	True	github.com/julienschmidt/httprouter
cfb55aafdaf3ec08f0db22699ab822c50091b1c4	True	github.com/kr/pretty
7cafcd837844e784b526369c9bce262804aebc60	True	github.com/kr/text
f4881e611bdbe9fb413a7780721ef8400a1f2341	True	github.com/lestrrat/go-jspointer
e452c7b5801d1c6494c9e7e0cbc7498c0f88dfd1	True	github.com/lestrrat/go-jsref
b09d7650b822d2ea3dc83d5091a5e2acd8330051	True	github.com/lestrrat/go-jsschema
b1258a10419fe0693f7b35ad65cd5074bc0ba1e5	True	github.com/lestrrat/go-jsval
2e6eaaa5717f81bda41d27070d3c966f40a1e75f	True	github.com/lestrrat/go-pdebug
f74c056fe41f860aa6264478c664a6fff8a64298	True	github.com/lestrrat/go-structinfo
4fbf7632a2c6d3fbdb9931439bdbbeded02cbe36	True	github.com/lunixbochs/vtclean
d0d66b4842f459c6c3e039a9e2f25eff02174ab1	True	github.com/lxc/lxd
ee4f0065d00cd12b542f18f5bc45799e88163b12	True	github.com/masterzen/azure-sdk-for-go
4572e39b1ab9fe03ee513ce6fc7e289e98482190	True	github.com/masterzen/simplexml
7a535cd943fccaeed196718896beec3fb51aff41	True	github.com/masterzen/winrm
13f4951698adc0fa9c1dda3e275d489a24201161	True	github.com/masterzen/xmlpath
ed8eb9e318d7a84ce5915b495b7d35e0cfe7b5a8	True	github.com/mattn/go-colorable
66b8e73f3f5cda9f96b69efd03dd3d7fc4a5cdb8	True	github.com/mattn/go-isatty
d96d1bd051f2bd9e7e43d602782b37b93b1b5666	True	github.com/mattn/go-runewidth
c12348ce28de40eed0136aa2b644d0ee0650e56c	True	github.com/matttproud/golang_protobuf_extensions
b2a7479cf26fa841ff90dd932d0221cb5c50782d	True	github.com/ncw/swift
179d4d0c4d8d407a32af483c2354df1d2c91e6c3	True	github.com/nu7hatch/gouuid
ad5c34ed0cf8169d6817e2a37ec3e4521f856368	True	github.com/oracle/oci-go-sdk
839d9e913e063e28dfd0e6c7b7512793e0a48be9	True	github.com/pkg/errors
575f371f7862609249a1be4c9145f429fe065e32	True	github.com/prometheus/client_golang
fa8ad6fec33561be4280a8f0514318c79d7f6cb6	True	github.com/prometheus/client_model
dd586c1c5abb0be59e60f942c22af711a2008cb4	True	github.com/prometheus/common
abf152e5f3e97f2fafac028d2cc06c1feb87ffa5	True	github.com/prometheus/procfs
6724a57986aff9bff1a1770e9347036def7c89f6	True	github.com/rogpeppe/fastuuid
5bf94b69c6b68ee1b541973bb8e1144db23a194b	True	github.com/satori/uuid
05504416e95561e1478196d7058721c827a7bb8c	True	github.com/vmware/govmomi
49796115aa4b964c318aad4f3084fdb41e9aa067	True	golang.org/x/crypto
61147c48b25b599e5b561d2e9c4f3e1ef489ca41	True	golang.org/x/net
a6bd8cefa1811bd24b86f8902872e4e8225f74c4	True	golang.org/x/oauth2
37707fdb30a5b38865cfb95e5aab41707daec7fd	True	golang.org/x/sys
b19bf474d317b857955b12035d2c5acb57ce8b01	True	golang.org/x/text
ed10e890a8366167a7ce33fac2b12447987bcb1c	True	google.golang.org/api
8c3190dff075bf5442c9eedbf8f8ed6144a099e7	True	gopkg.in/amz.v3
4f90aeace3a26ad7021961c297b22c42160c7b25	True	gopkg.in/check.v1
442357a80af5c6bf9b6d51ae791a39c3421004f3	True	gopkg.in/errgo.v1
9cc2ac8f3ac3904725fe9bef85d30b53713b3be9	True	gopkg.in/goose.v2
1a21782420ea13c3c6fb1d03578f446b3248edb1	True	gopkg.in/httprequest.v1
776aa739ce9373377cd16f526cdf06cb4c89b40f	True	gopkg.in/ini.v1
51fa6e26128d74e445c72d3a91af555151cc3654	True	gopkg.in/juju/blobstore.v2
abaa717bc6ed4f694281737cb01f32f7de7ea357	True	gopkg.in/juju/charm.v6
653bbd81990d2d7d48e0fb931a3b0f52c694572f	True	gopkg.in/juju/charmrepo.v2
7778a447283bd71109671c20818544514e16e9d9	True	gopkg.in/juju/charmrepo.v3
743b5ab3209d901b82aaa6594c405e017e0761a9	True	gopkg.in/juju/charmstore.v5
7359fc7857abe2b11b5b3e23811a9c64cb6b01e0	True	gopkg.in/juju/environschema.v1
1ebf5c5481e815de683e57e0f46705ac567e4e58	True	gopkg.in/juju/jujusvg.v3
fd59336b4621bc2a70bf96d9e2f49954115ad19b	True	gopkg.in/juju/names.v2
6965b9d826717287bb002e02d1fd4d079978083e	True	gopkg.in/juju/worker.v1
469b44e6f1f9479e115c8ae879ef80695be624d5	True	gopkg.in/macaroon-bakery.v1
ec9d2ad6796100720c154f614b6dea8798ec1181	True	gopkg.in/macaroon-bakery.v2
5a131df02b2333d5d75c501743cbc2948ee9bbf0	True	gopkg.in/macaroon-bakery.v2-unstable
ab3940c6c16510a850e1c2dd628b919f0f3f1464	True	gopkg.in/macaroon.v1
bed2a428da6e56d950bed5b41fcbae3141e5b0d0	True	gopkg.in/macaroon.v2
66ab28d0d56f39a383f7cf5cf3ac9a4e9ab32865	True	gopkg.in/macaroon.v2-unstable
f2b6f6c918c452ad107eec89615f074e3bd80e33	True	gopkg.in/mgo.v2
df99d62fd42d8b3752c8a42c6723555372c02a03	True	gopkg.in/natefinch/lumberjack.v2
c1b8fa8bdccecb0b8db834ee0b92fdbcfa606dd6	True	gopkg.in/natefinch/npipe.v2
2d7c7c65cc71d024968d9ff4385d5e7ad3a83fcc	True	gopkg.in/retry.v1
dd632973f1e7218eb1089048e0798ec9ae7dceb8	True	gopkg.in/tomb.v1
14b3d72120e8d10ea6e6b7f87f7175734b1faab8	True	gopkg.in/tomb.v2
1be3d31502d6eabc0dd7ce5b0daab022e14a5538	True	gopkg.in/yaml.v2
183f3326a9353bd6d41430fc80f96259331d029c	True	k8s.io/api
cbafd24d5796966031ae904aa88e2436a619ae8a	True	k8s.io/apimachinery
3db81bdd128696db1c2fcba8a426ed0a3993824d	True	k8s.io/client-go

Reply:
dependencies.tsv hasn't had any new upstream repos added this week, but the existing upstream repos have been updated, including worker, errors, testing

Reply:
just synced with current develop branch.